Peer.Domain/Formatters/CompactFormatter.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Peer.Domain.Util;
+
+namespace Peer.Domain.Formatters;
+
+//q(cn): Is there just an object that represents the parts of the format? 
+//  if we're talking format it's include/exclude different properties and max
+//  widths...
+//var x = new
+//{
+//    Id = ("Id", 0, 4, true),
+//    Title = ("Title", 1, 40, true),
+//    Status = ("S", 2, 1, true),
+//    Url = ("Url", 3, 35, true)
+//}; or similar sort of structure?
+public class CompactFormatter : IPullRequestFormatter
+{
+    private readonly ISymbolProvider _symbolProvider;
+    private const char _ellipsis = '\u2026';
+    private static readonly string _header = $"{"Id",-4} {"Title",-40} {"\ud83d\udea7"} Url";
```

Comment:
Named constant?

Reply:
I meant for the Unicode, but I wonder if you can do it for the lengths too

Reply:
Is \ud83d\udea7 a roadblock?

Reply:
Yeah